geppetto-client/js/components/interface/3dCanvas/OBJLoader.js

```diff
@@ -444,7 +444,7 @@ THREE.OBJLoader.prototype = {
     var material = new THREE.PointsMaterial(
       {
         size: 2,
-        map: textureLoader.load("geppetto/node_modules/@geppettoengine/geppetto-client/js/components/interface/3dCanvas/particle.png"),
+        map: textureLoader.load("geppetto/node_modules/@geppettoengine/geppetto-client/geppetto-client/js/components/interface/3dCanvas/particle.png"),
```

Comment:
Those `geppetto-client/geppetto-client` scare me. If someone looks at it without knowing the history easily thinks thinks it's a bad copy/paste to fix. Also we should remove direct references to files being inside node_modules. If we don't do that, we need to include the node_modules folder in every deployment. @tarelli I think that getting rid of those references may be one of the next refactorings.